bin/configs/kotlin-oneOf-discriminator-kotlinx-serialization.yaml

```diff
@@ -0,0 +1,12 @@
+generatorName: kotlin
+outputDir: samples/client/others/kotlin-oneOf-discriminator-kotlinx-serialization
```

Comment:
please add the new folder to the gihtub workflow so that it will be tested moving forward

https://github.com/OpenAPITools/openapi-generator/blob/master/.github/workflows/samples-kotlin-client.yaml